packages/suite/src/actions/suite/__fixtures__/metadataActions.ts

```diff
@@ -536,6 +538,36 @@ const init = [
     },
 ];
 
+const getLabelableEntitiesDescriptors = [
```

Comment:
I prefer to type everything, event fixtures declaratively. 

1) Allows IDE auto-completion
2) When type changes it will raise **ŧype error** here and you dont need to debug failing test.